ui/src/app/components/cromwell-configuration-panel.spec.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+
+import {
+  DisksApi,
+  ProfileApi,
+  WorkspaceAccessLevel,
+  WorkspacesApi,
+} from 'generated/fetch';
+import { AppsApi } from 'generated/fetch/api';
+
+import { Spinner } from 'app/components/spinners';
+import {
+  appsApi,
+  profileApi,
+  registerApiClient,
+} from 'app/services/swagger-fetch-clients';
+import { currentWorkspaceStore } from 'app/utils/navigation';
+import { profileStore, serverConfigStore } from 'app/utils/stores';
+
+import defaultServerConfig from 'testing/default-server-config';
+import {
+  mountWithRouter,
+  waitOneTickAndUpdate,
+} from 'testing/react-test-helpers';
+import { AppsApiStub } from 'testing/stubs/apps-api-stub';
+import { CdrVersionsStubVariables } from 'testing/stubs/cdr-versions-api-stub';
+import { DisksApiStub } from 'testing/stubs/disks-api-stub';
+import { ProfileApiStub } from 'testing/stubs/profile-api-stub';
+import {
+  workspaceStubs,
+  WorkspaceStubVariables,
+} from 'testing/stubs/workspaces';
+import { WorkspacesApiStub } from 'testing/stubs/workspaces-api-stub';
+
+import { defaultCromwellConfig } from './apps-panel/utils';
+import { CromwellConfigurationPanel } from './cromwell-configuration-panel';
+
+interface Props {
+  onClose: () => void;
+}
+
+describe('CromwellConfigurationPanel', () => {
+  let props: Props;
+  let disksApiStub: DisksApiStub;
+  let workspacesApiStub: WorkspacesApiStub;
+  let onClose: () => void;
+  let freeTierBillingAccountId: string;
+
+  const component = async (propOverrides?: object) => {
+    const allProps = { ...props, ...propOverrides };
+    const c = mountWithRouter(<CromwellConfigurationPanel {...allProps} />);
+    await waitOneTickAndUpdate(c);
+    return c;
+  };
```

Comment:
I really like "default props + overrides" functions in react tests like this 👍 

One minor suggestion would be to define a `DEFAULT_PROPS` constant to have only one way to define overridden props (currently you could define `propOverrides` or mutate `props`). I personally find it easier to understand at a glance than diving into `beforeEach`.